configure.ac

```diff
@@ -34,6 +34,14 @@ dnl Checks for programs
 AC_PROG_CC
 AM_PROG_CC_C_O
 
+AC_ARG_WITH([libfabric],
+            AC_HELP_STRING([--with-libfabric], [Use non-default libfabric location - default NO]),
+            [AS_IF([test -d $withval/lib64], [fab_libdir="lib64"], [fab_libdir="lib"])
+             [CPPFLAGS="$CPPFLAGS -I $withval/include"]
+             [LIBS="$LIBS -lfabric"]
```

Comment:
You don't need this LIBS line.  LIBS will get -lfabric added at the AC_CHECK_LIB on line 46.